src/uct/ib/base/ib_iface.c

```diff
@@ -1173,6 +1174,87 @@ static void uct_ib_iface_set_path_mtu(uct_ib_iface_t *iface,
     }
 }
 
+const char*
+uct_ib_iface_sl_mask_str(uint16_t sl_mask, ucs_string_buffer_t *str_b)
```

Comment:
maybe put in ucs/string.c: 'ucs_status_t ucs_mask_str(uint64_t mask, ucs_string_buffer_t *strb)'